File: rb/test/exists_test.rb
# RealtimeWeather SDK exists test

require "minitest/autorun"
require_relative "../RealtimeWeather_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = RealtimeWeatherSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- RealtimeWeather SDK exists test

local sdk = require("realtime-weather_sdk")

describe("RealtimeWeatherSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
